cosmoz-grouped-list.js

```diff
@@ -456,6 +464,23 @@
 			}
 		},
 
+		highlightItem(item, reverse) {
+			var model = this._getModelFromItem(item),
+				highlightedIndex = this.highlightedItems.indexOf(item);
+
+			if (!this.isItemHighlighted(item) && !reverse) {
```

Comment:
Maybe it's not necessary to call `isItemHighlighted` as this will call indexOf on highlighted items again.